### DISCUSSION TITLE
only specific cards to dual mining eth + ton

### DISCUSSION THREAD

**Discussion by mkuttert** [Lolliedieb/lolMiner-releases#1136] [Q&A]
good night, how do I select only specific cards to do dual mining eth + ton?

**Comment by eaamgh** [comment]
It needs v1.44 then use TONDUAL mode along with --dualdevices switch

**Reply by cypher619** [reply]
--maxdualimpact does this not --dualdevices see Guide Tuning Dual Mining in the Wiki section
